app/views/organisations/_show_organisation.html.erb

```diff
@@ -130,6 +130,21 @@
   <% end %>
 <% end %>
 
+<% if @show.high_profile_groups[:items] %>
+  <div class="grid-row organisations__section">
```

Comment:
We should probably start separating out these sections into partials (although this chunk is quite small).

Reply:
@andysellick Yep, probably one for a tidy-up PR near the end